src/page-modules/departures/layout.tsx

```diff
@@ -65,11 +65,12 @@ function DeparturesLayout({ children, fromQuery }: DeparturesLayoutProps) {
           transition={{ duration: 0.25, ease: [0.04, 0.62, 0.23, 0.98] }}
           className={style.main}
         >
+          <TabLink activePath={'/departures'} />
```

Comment:
The template literal syntax and `{}` is in this case just syntactic noise. Should try to always use regular strings directly when possible
```suggestion
          <TabLink activePath="/departures" />
```